homeassistant/components/roborock/strings.json

```diff
@@ -428,6 +428,20 @@
     "get_maps": {
       "name": "Get maps",
       "description": "Get the map and room information of your device."
+    },
+    "vacuum_goto": {
+      "name": "Vacuum go to",
+      "description": "Go to the specified coordinates.",
+      "fields": {
+        "x_coord": {
+          "name": "X coordinate",
+          "description": "X-coordinate."
+        },
+        "y_coord": {
+          "name": "Y coordinate",
+          "description": "Y-coordinate."
+        }
```

Comment:
How does one find out what coordinate to pick?

Reply:
We could potentially do a sensor that says its current x y coordinates? Or maybe add something to the image component? Otherwise I think just guessing or checking

Reply:
I think we can do 2 entities for that, disabled by default

Reply:
That probably can't be done until we get #112047 done.

That is a refactor of the image component and then some knowledge needs to be extracted out of there. Unfortunately, the only way to get the X y coordinates is from the map (that I'm aware of). So the logic of getting the map will have to be moved out of the image component and into the coordinator.

Reply:
A service or sensor for the current position would definitely be an advantage. I managed to get the position using the app and by executing the action, but I would initially base my documentation on this: https://www.home-assistant.io/integrations/xiaomi_miio/#action-xiaomi_miiovacuum_goto

Reply:
due to the complexity I would separate this from this pr

Reply:
added docs pr https://github.com/home-assistant/home-assistant.io/pull/36554